server/cmake/modules/FindLibClang.cmake

```diff
@@ -16,7 +16,7 @@
 # most recent versions come first
 # http://llvm.org/apt/
 set(LIBCLANG_KNOWN_LLVM_VERSIONS 5.0.0 5.0
```

Comment:
Could you show the content of `/usr/lib{,64}/cmake/clang/ClangConfig.cmake`? Not sure about the path, a more violent solution could be to use `find / -name ClangConfig.cmake` or something along these lines.

The content is not supposed to be too big.